cmds/core/ip/ops.go

```diff
@@ -130,3 +131,66 @@ func showNeighbours(w io.Writer, withAddresses bool) error {
 	}
 	return nil
 }
+
+const (
+	defaultFmt = "default via %v dev %s proto %s metric %d\n"
+	routeFmt   = "%v dev %s proto %s scope %s src %s metric %d\n"
+)
+
+var rtProto = map[int]string{
```

Comment:
can you add a comment that says where are these values coming from? I know they are in somewhere in iproute2's source code, but if there's an authoritative source (kernel docs?) that's even better

Reply:
i found some references in manpages :+1: